src/test/java/SetTest.java

```diff
@@ -0,0 +1,47 @@
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.CsvSource;
+import org.junit.jupiter.params.provider.ValueSource;
+
+import java.util.HashSet;
+import java.util.Set;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.junit.jupiter.api.Assertions.*;
+public class SetTest {
+    private Set<Integer> numbers;
+
+    @BeforeEach
+    void setUp() {
+        numbers = new HashSet<>();
+        numbers.add(1);
+        numbers.add(1);
+        numbers.add(2);
+        numbers.add(3);
+    }
+
+    @Test
+    @DisplayName("size() 메소드를 활용해 Set 크기를 확인한다.")
+    void checkSetSize() {
+        int setSize = numbers.size();
+        assertThat(setSize).isEqualTo(3);
```

Comment:
```suggestion
        assertThat(numbers).hasSize(3);
```
컬렉션을 테스트할 수 있도록 제공되는 메소드가 있으니 적극적으로 활용하면 좋겠네요

Reply:
아 컬렉션 테스트 메소드가 있었네요! 잘 찾아보며 진행하겠습니다